starport/pkg/sdkerror/error.go

```diff
@@ -0,0 +1,30 @@
+package sdkerror
+
+import (
+	"errors"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var (
+	ErrSPNInternal    = errors.New("some invariants expected by the underlying system has been broken")
+	ErrInvalidRequest = errors.New("invalid GRPC request argument or object not found")
```

Comment:
I actually don't know if `invalid GRPC request argument or object not found` is the right wording. This is the message printed for any message where the handler fails.
So maybe it could be confusing the error, for example, the that a request is sent the a chain where the launch has already been triggered (the request argument is actually right since the chain exist, the error is related to the internal chain state)
The is open for debate maybe we can consider that the argument is invalid because we provide a chain that has been already launched
This is not blocking for the PR

Reply:
but with the `sdkerror.Unwrap` we can get the properly chain errors and compare like:

```go
err = sdkerror.Unwrap(err)
if err == launchtypes.ErrTriggeredLaunch {
    // TODO handle already triggered launch error
}
```